The musl resolver does not define res_ninit() and res_nclose() functions
like glibc does. A wrapper for musl to mimick GLIBC's function should do
the trick.

--- qtwebengine/src/3rdparty/chromium/net/base/dns_reloader.cc	2015-06-03 12:34:26.979892244 +0200
+++ qtwebengine/src/3rdparty/chromium/net/base/dns_reloader.cc	2015-06-03 12:32:07.091899808 +0200
@@ -8,6 +8,9 @@
     !defined(OS_ANDROID)

 #include <resolv.h>
+#if defined(OS_LINUX) && !defined(__GLIBC__)
+#include "net/dns/resolv_compat.h"
+#endif

 #include "base/basictypes.h"
 #include "base/lazy_instance.h"
--- qtwebengine/src/3rdparty/chromium/net/dns/dns_config_service_posix.cc	2015-02-17 05:58:45.000000000 +0100
+++ qtwebengine/src/3rdparty/chromium/net/dns/dns_config_service_posix.cc	2015-06-04 22:22:01.104300355 +0200
@@ -21,6 +21,10 @@
 #include "net/dns/notify_watcher_mac.h"
 #include "net/dns/serial_worker.h"

+#if defined(OS_LINUX) && !defined(__GLIBC__)
+#include "net/dns/resolv_compat.h"
+#endif
+
 #if defined(OS_MACOSX) && !defined(OS_IOS)
 #include "net/dns/dns_config_watcher_mac.h"
 #endif
